src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/DeviceCulture/DeviceCulture.ios.macos.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using Foundation;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class DeviceCulture
+	{
+		private static partial IReadOnlyList<string> GetPreferredCultureStrings() =>
+			NSLocale.PreferredLanguages;
+
+		private static partial CultureInfoParser GetCultureParser() => new iOSCultureInfoParser();
+
+		class iOSCultureInfoParser : CultureInfoParser
+		{
+			protected override string ToDotnetFallbackLanguage(string languageCode)
+			{
+				languageCode = base.ToDotnetFallbackLanguage(languageCode);
+
+				var netLanguage = languageCode switch
+				{
+					"pt" => "pt-PT", // Fallback to Portuguese (Portugal)
```

Comment:
There are much more such cases

Reply:
Do you know some examples? If so, I would be happy to add them. This method is meant to be expanded (as written in it's description)

Reply:
ar_EG_#u-nu-lat

Reply:
This method only processes language code (there is different one for language + region). I think `ar` should be supported in .net. 
If `ar_EG` is not supported, could you please tell closest equivalent in .net?